l10n_it_pec_messages/model/mail_thread.py

```diff
@@ -212,3 +295,20 @@ def _message_find_partners_pec(
             if partner_ids:
                 res = partner_ids[0]
         return res
+
+    def _FindOrCreatePartnersPec(
```

Comment:
questo metodo nella #15 era usato in parse_daticert anche perche' pec_messages di base quando apre il wizard per rispondere ad una pec, cerca un partner e quindi se non lo trova si spacca.